xbmc/listproviders/DirectoryProvider.cpp

```diff
@@ -11,12 +11,19 @@
 #include "ContextMenuManager.h"
 #include "FileItem.h"
 #include "ServiceBroker.h"
+#include "ThumbLoader.h"
+#include "addons/AddonEvents.h"
+#include "addons/AddonManager.h"
 #include "addons/gui/GUIDialogAddonInfo.h"
+#include "commons/ilog.h"
```

Comment:
This is also strange

Reply:
It's pulled in for the defines for LOG_DEBUG, LOG_INFO, etc.
If you want a symbol mapping to something else, just let me know what header you think it should be, and I'll add that to the remapping

Reply:
well this should just be pulled in through `utils/log.h` if not we should just remove `ilog.h` and move it's contents to `utils/log.h`